externs/browser/w3c_indexeddb.js

```diff
@@ -572,35 +471,52 @@ IDBIndex.prototype.keyPath;
  * @type {boolean}
  * @const
  */
+IDBIndex.prototype.multiEntry;
+
+/**
+ * @type {boolean}
+ * @const
+ */
 IDBIndex.prototype.unique;
 
 /**
- * @param {IDBKeyRange=} range The range of the cursor.
- * @param {(number|string)=} direction The direction of cursor enumeration.
+ * TODO(vobruba-martin): Change direction param to !IDBCursorDirection once CC understands string literal types.
```

Comment:
Please make your to-dos just "TODO: ..."
Unfortunately non-googler names in TODOs break internal tools.